backend/src/repos/registerData.js

```diff
@@ -0,0 +1,20 @@
+import {db} from '../data/connection';
+
+const registerQuery = {
+    user : 'INSERT INTO user (name,password) VALUES(?,?)',
+    kingdom : 'INSERT INTO kingdom (name) VALUES(?)',
+    user_kingdom : 'INSERT INTO user_kingdom (user_id,kingdom_id) VALUES(?,?)'
+};
+
+export const registerData = (table,params) => {
+    console.log(params)
+    return new Promise ( async (resolve,reject) => {
+        try {
+            const returnData = (await db.query(registerQuery[table],params,"insertId")).specialReturn;
```

Comment:
@gulyaasjaanos didn't you revert this `specialReturn` since it is returned anyway?

Reply:
this file shouldn't be here at all